stdlib/tkinter/__init__.pyi

```diff
@@ -384,8 +389,32 @@ class Misc:
     @overload
     def grid_bbox(self, column: int, row: int, col2: int, row2: int) -> Tuple[int, int, int, int] | None: ...
     bbox = grid_bbox
-    def grid_columnconfigure(self, index, cnf=..., **kw): ...  # TODO
-    def grid_rowconfigure(self, index, cnf=..., **kw): ...  # TODO
+    # The cnf parameter's default value is an empty dict. That's a perfectly
+    # valid instance of _GridIndexInfo, but MyPy declares that an empty dict
+    # literal must be dict[Any, Any] and is therefore incompatible.
```

Comment:
This is correct, but a bit surprising to me. Compare it with this for example:

```python
from typing import TypedDict

class _GridIndexInfo(TypedDict, total=False):
    weight: int

def foo(x: _GridIndexInfo):
    pass

foo({})  # No error, mypy knows it's a TypedDict
bar = {}
foo(bar)  # Error
```

I'm a bit surprised that default parameters somehow don't work this way.